Makefile

```diff
@@ -1,3 +1,5 @@
+.NOTPARALLEL:
```

Comment:
from dev meeting: change place to not be the default target & add comment to higlight that it is a quick fix

Reply:
Done